### PR TITLE
Extend module output with `node_id`

### PR DESCRIPTION
**Pull request by snovikov**

### CODE REVIEW

File: modules/team/outputs.tf [hunk 1]
```diff
@@ -8,6 +8,11 @@ output "id" {
   value       = github_team.team.id
 }
 
+output "node_id" {
+  description = "GitHub team node_id"
```
Comment (cytopia) [inline modules/team/outputs.tf:12]: What is the node_id? Description doesn't really tell

Comment (snovikov) [reply modules/team/outputs.tf:12]: This is internal hashed id used by GitHub internally. There is no information regarding this in the provider: https://registry.terraform.io/providers/integrations/github/latest/docs/resources/team#node_id

But you can get some hints about it here: https://docs.github.com/en/graphql/guides/using-global-node-ids

Comment (cytopia) [inline modules/team/outputs.tf:12]:
```suggestion
  description = "This is internal hashed id used by GitHub internally for teams."
```

File: modules/team/README.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ No modules.
 |------|-------------|
 | <a name="output_name"></a> [name](#output\_name) | GitHub team name |
 | <a name="output_id"></a> [id](#output\_id) | GitHub team id |
+| <a name="output_node_id"></a> [node\_id](#output\_node\_id) | GitHub team node\_id |
```
Comment (cytopia) [inline modules/team/README.md:44]:
```suggestion
| <a name="output_node_id"></a> [node\_id](#output\_node\_id) | his is internal hashed id used by GitHub internally for teams. |
```

**Review by cytopia — APPROVED** [review]

**Pull request merged**